### DISCUSSION TITLE
Windows: libmp3lame.DLL was not found during installation

### DISCUSSION THREAD

**Discussion by jonasbue** [monocasual/giada#664] [Q&A] [answered]
The Windows installer (Windows 11) stopped because the libmp3lame.DLL file is missing/not found.

How do I approach this? Is it a missing dependency I need to install myself?
Thanks for any help :)

**Comment by jmeadows1987** [comment]
I am having the same issue.  Looks like it has popped up with other audio software as well (someone had the issue with Reason).  I am running windows 11 pro, I see lots of very unverifiable sites to download *.dll files from, but would like to hear an "official" answer first

**Reply by gvnnz** [reply, marked as answer]
Hey @jonasbue and @jmeadows1987 , this issue has been fixed today :sunglasses: https://github.com/monocasual/giada/issues/661
And will be included in version 0.26. Thank you!

**Comment by bluntrapture** [comment]
**I have been having the SAME issue: missing libmp3lame.dll  --- and NOTHING I've found (or understand) works to fix it.

Then I finally came across this discussion, and gnnz says it's been fixed.

Then WHY am I having this same problem?

I'm very close to losing my mind.**

**Reply by gvnnz** [reply]
Please see #668, we fixed it in the latest release.

**Comment by bluntrapture** [comment]
THANK YOU!
    On Monday, October 16, 2023 at 01:49:53 AM PDT, Giovanni A. Zuliani ***@***.***> wrote:  
 
 


Please see #668, we fixed it in the latest release.
